test/helpers/utils.go

```diff
@@ -147,6 +150,61 @@ func WithTimeoutErr(ctx context.Context, f func() (bool, error), freq time.Durat
 	}
 }
 
+// InstallExampleCilium uses Cilium Kubernetes example from the repo,
+// changes the etcd parameter and installs the stable tag from docker-hub
+func InstallExampleCilium(kubectl *Kubectl) {
+
+	var path = "../examples/kubernetes/cilium.yaml"
+	var result bytes.Buffer
+	timeout := time.Duration(300)
+
+	newCiliumDSName := fmt.Sprintf("cilium_ds_%s.json", MakeUID())
+
+	objects, err := DecodeYAMLOrJSON(path)
+	Expect(err).To(BeNil())
+
+	for _, object := range objects {
+		data, err := json.Marshal(object)
+		Expect(err).To(BeNil())
+
+		jsonObj, err := gabs.ParseJSON(data)
+		Expect(err).To(BeNil())
+
+		value, _ := jsonObj.Path("kind").Data().(string)
+		if value == configMap {
+			jsonObj.SetP("---\nendpoints:\n- http://k8s1:9732\n", "data.etcd-config")
+			jsonObj.SetP("true", "data.debug")
+		}
+
+		result.WriteString(jsonObj.String())
```

Comment:
TIL that simply putting json objects one after another work in json K8s manifests.